src/Components/Reviews.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ReviewCard from "./ReviewCard";
+
+export default function Reviews({ reviews }) {
+  return (
+    <div className="Reviews flex flex-wrap gap-4">
+      {reviews.map((review) => {
+        return <ReviewCard key={crypto.randomUUID()} {...review} />;
```

Comment:
interesting! Is there any data we already have access to in `review` that's unique to each review? Remember keys don't need to be unique across the entire app so in this case a review id would be fine

Reply:
cool, updated to review.review_id